pkg/api/const.go

```diff
@@ -319,12 +319,11 @@ const (
 	// AzureCniPluginVerLinux specifies version of Azure CNI plugin, which has been mirrored from
 	// https://github.com/Azure/azure-container-networking/releases/download/${AZURE_PLUGIN_VER}/azure-vnet-cni-linux-amd64-${AZURE_PLUGIN_VER}.tgz
 	// to https://kubernetesartifacts.azureedge.net/azure-cni
-	// AKS needs 1.0.33 for node local dns cache support
-	AzureCniPluginVerLinux = "v1.0.33"
+	AzureCniPluginVerLinux = "v1.1.3"
 	// AzureCniPluginVerWindows specifies version of Azure CNI plugin, which has been mirrored from
 	// https://github.com/Azure/azure-container-networking/releases/download/${AZURE_PLUGIN_VER}/azure-vnet-cni-windows-amd64-${AZURE_PLUGIN_VER}.zip
 	// to https://kubernetesartifacts.azureedge.net/azure-cni
-	AzureCniPluginVerWindows = "v1.0.33"
+	AzureCniPluginVerWindows = "v1.1.3"
```

Comment:
@AbelHu Does the current windows VHD contains this CNI?

Reply:
no. Only contains v1.0.33, v1.1.0 and v1.1.2

Reply:
I was assuming this would get updated in aks-engine master and I could cherry pick it over the last week.  But looks like it's been 21 days sinceupdate. 
https://github.com/Azure/aks-engine/commit/0914ac1be68fb77dfa941e9b2be857dec373afc4

Figuring out who  I need to talk to.

Reply:
Jack is updating today. 
https://teams.microsoft.com/l/message/19:3f80230d87644cbf8e56d5c15187aaee@thread.skype/1591038043608?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=e121dbfd-0ec1-40ea-8af5-26075f6a731b&parentMessageId=1591038043608&teamName=Azure%20Container%20Compute&channelName=AKS%20Engine&createdTime=1591038043608

Reply:
Cherry picked jack's change fda4cb4e0cb7fcec75598e770b1d89fa7a8b83dd.

Reply:
@xuto2 I think this is ready. Let me know if I need to vendor my self or if I should just ride something about to be done.